rkt/api_service.go

```diff
@@ -289,26 +283,27 @@ func getNetworks(p *pod) []*v1alpha.Network {
 func getBasicPod(p *pod) (*v1alpha.Pod, *schema.PodManifest, error) {
```

Comment:
@euank I don't feel a proper state that we call it `POD_STATE_UNDEFINED`, because we actually know the state of the pod, it just lacks some information.